recipes/libcpuid/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "libcpuid" %}
+{% set version = "0.5.1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/anrieff/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 36d62842ef43c749c0ba82237b10ede05b298d79a0e39ef5fd1115ba1ff8e126
+
+build:
+  number: {{ build }}
```

Leave this without a template, otherwise the bot may hiccup over this.
```suggestion
  number: 0
```